### ISSUE TITLE
Empty page when accessing host details

### ISSUE THREAD

**Issue by ampc** [hashicorp/boundary#3170]
**Describe the bug**
When opening host details from an entry created through host catalogue, a blank page is displayed.

**To Reproduce**
1. Create a Host Catalog using Azure plugin
2. Create Host Set
3. Open Host page
![image](https://user-images.githubusercontent.com/284907/233377180-d25c6c2a-7174-40a4-a8eb-fe128e9ca73e.png)

**Expected behavior**
Display host details page instead of blank page.

**Additional context**
When obtaning details through cli, it works
```shell
boundary hosts read -id hplg_RUJpV77KjF

Host information:
  Created Time:        Thu, 20 Apr 2023 14:05:20 WEST
  External ID:         /subscriptions/...
  Host Catalog ID:     hcplg_dsbMtXJ1GO
  ID:                  hplg_RUJpV77KjF
  Type:                plugin
  Updated Time:        Thu, 20 Apr 2023 14:05:20 WEST
  Version:             1

  Scope:
    ID:                p_1234567890
    Name:              Generated project scope
    Parent Scope ID:   o_1234567890
    Type:              project

  Plugin:
    ID:                pl_8MJZDaQv8R
    Name:              azure

  Authorized Actions:
    no-op
    read

  Host Set IDs:
    hsplg_QoLIYcmkpH

  IP Addresses:
    10.12.1.9
    72.X.Y.Z
 ```

**Comment by randallmorey** [comment]
Hi there and thanks for your interest in Boundary.  Can you provide additional information regarding any errors from the browser console and network requests from the browser network tab relevant to this route?

**Comment by ampc** [comment]
@randallmorey 
Sure, I only have this
![image](https://user-images.githubusercontent.com/284907/233432627-b7c656f9-1557-416a-a712-ccb21988a1fe.png)

**Comment by xingluw** [comment]
Likely fixed in 0.13 release as it cannot be reproduced.
